src/main/java/com/sap/broker/budgie/resources/api/CatalogResource.java

```diff
@@ -1,28 +1,46 @@
 package com.sap.broker.budgie.resources.api;
 
 import javax.inject.Inject;
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
+import javax.ws.rs.*;
 import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
 
 import com.sap.broker.budgie.domain.Catalog;
+import com.sap.broker.budgie.domain.Service;
+import com.sap.broker.budgie.domain.Visitor;
 import com.sap.broker.budgie.impl.ServiceBroker;
 
-@Path("/catalog")
+import java.util.List;
+
+@Path("/{configId}/v2/catalog")
```

Comment:
Isnt't that the spaceId and not configId?

Reply:
the space name is used in the OQ tests because of its uniqueness across the tests